packages/statemanager/examples/evm.ts

```diff
@@ -0,0 +1,15 @@
+import { RPCStateManager, RPCBlockChain } from '@ethereumjs/statemanager'
+import { EVM } from '@ethereumjs/evm'
+
+const main = async () => {
+  try {
+    const provider = 'https://path.to.my.provider.com'
+    const blockchain = new RPCBlockChain(provider)
+    const blockTag = 1n
+    const state = new RPCStateManager({ provider, blockTag })
+    const evm = new EVM({ blockchain, stateManager: state })
+  } catch (e) {
+    console.log(e)
```

Comment:
Maybe log just the error message here and add some comments in the example explaining why it failed (i.e. `fetch should fail unless you replace with a real provider url string`)

Reply:
Fixed